understanding/understanding-techniques.html

```diff
@@ -229,150 +209,134 @@ <h3>Numbered Lists, "AND"</h3>
                         					
-                        <p>they may not be testable;</p>
+                        <p>テスト可能でない。</p>
                         				
                      </li>
                      				
                      <li>
                         					
-                        <p>in some circumstances they may not be applicable or practical, and may even decrease
-                           accessibility for some users while increasing it for others;
+                        <p>ある状況においてそのテクニックが適用可能でないあるいは実用的でなく、かつその他の利用者に対してはアクセシビリティを向上させる一方、一部の利用者に対してアクセシビリティを低下させる可能性がある。
                         </p>
                         				
                      </li>
                      				
                      <li>
                         					
-                        <p>they may not address the success criterion itself, and instead provide related accessibility
-                           benefits.
+                        <p>達成基準そのものには対処しておらず、代わりに関連するアクセシビリティ上の利点を提供している。
                         </p>
                         				
                      </li>
                      			
                   </ul>
                   			
-                  <p>Authors are encouraged to apply all of the techniques where appropriate to best address
-                     the widest range of users' needs.
+                  <p>コンテンツ制作者には、最も幅広い利用者のニーズに対処するのに最適なすべてのテクニックを適用することが推奨される。
                   </p>
                   		
                </section>
                		
                <section id="failures">
                   			
-                  <h2>Failures</h2>
+                  <h2>失敗例</h2>
                   			
-                  <p><em>Failures</em> are things that cause accessibility barriers and fail specific success criteria.
-                     The documented <em>failures</em> are useful for:
+                  <p><em>失敗例</em>はアクセシビリティ上のバリアを引き起こし、特定の達成基準への不適合を招くものである。文書化された<em>失敗例</em>は以下の点で有益である:
                   </p>
                   			
                   <ul>
                      				
                      <li>
                         					
-                        <p>Authors to know what to avoid,</p>
+                        <p>コンテンツ制作者にとっては、避けるべきことを知ることができる。</p>
                         				
                      </li>
                      				
                      <li>
                         					
-                        <p>Evaluators to use for checking if content does not meet WCAG success criteria.</p>
+                        <p>評価者にとっては、コンテンツが WCAG 2.2 の達成基準を満たしていない場合に、チェックに用いることができる。</p>
                         				
                      </li>
                      			
                   </ul>
                   			
-                  <p>Content that has a <em>failure</em> does not meet WCAG success criteria, unless an alternate version is provided without
-                     the failure.
+                  <p><em>失敗例</em>にあるコンテンツは、失敗例にない代替版が提供されていなければ、WCAG 2.2 の達成基準を満たしていない。
```

Comment:
「失敗例にあるコンテンツ」と言われると失敗例の文書に書かれているコンテンツのようにも読めますが、ここでは、あるコンテンツが失敗の条件を満たしてしまっている、という意味かと思います。
ここの failure も「失敗例」ではなく「失敗」とすべきものかもしれません。
こんな感じでしょうか?

```suggestion
                  <p><em>失敗</em>があるコンテンツは、失敗のない代替版が提供されていない限り、WCAG 2.2 の達成基準を満たさない。
```

Reply:
なるほど。
「失敗のない」と後で来ているので「失敗のある」として取り込みました。